packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -157,6 +169,36 @@ class Metrics {
     return payload;
   }
 
+  initMediaPayload(eventType, identifiers, options) {
+    const payload = {
+      eventId: uuid.v4(),
+      version: 1,
+      origin: {
+        buildType: 'prod',
```

Comment:
Does this need to change when run on integration?

Reply:
No, it only accepts prod as of now